src/server.cc

```diff
@@ -4,3 +4,17 @@
 // file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
 #include "server.h"
+
+void Server::do_accept()
+{
+  acceptor_.async_accept(socket_,
+      [this](boost::system::error_code ec)
+      {
```

Comment:
Can we update the formatting to have tabs for scope? L12-19 is in the same tab region and for readability it should probably have some indentation.